raiden/raiden_service.py

```diff
@@ -335,10 +337,12 @@ def stop(self):
         # We need a timeout to prevent an endless loop from trying to
         # contact the disconnected client
         try:
-            with gevent.Timeout(self.shutdown_timeout):
+            with gevent.Timeout(timeout, 'Substasks didn\'t stopped on time'):
+                self.transport.stop()
+                self.alarm.stop()
                 self.blockchain_events.uninstall_all_event_listeners()
         except (gevent.timeout.Timeout, RaidenShuttingDown):
-            pass
+            log.exception('Exception on RaidenService stop')
```

Comment:
`RaidenShuttingDown`exceptions are expcted durring shutting down and should not be logged.

Reply:
I hope `RaidenShuttingDown` won't be needed anymore after proper `stop` and exception handling here in this PR, but no hard feelings for it too